test/tiny/test-save-partial-proof.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+IGNORE=$($KORE_REPL test-a-to-c-vdefinition.kore -r --repl-script run-one-save-state-repl-script-spec.k.repl --module VERIFICATION --prove test-a-to-c-spec.kore --spec-module A-TO-C-SPEC)
```

Comment:
Why the `IGNORE=$(...)` part?

Reply:
The Makefile will dump all the output from the script into the `.golden` and `.out` files, and I wanted to ignore the output from the two repl processes since I'm only interested in `diff`-ing the config files. I tried to do that by redirecting the outputs to `/dev/null` (which seemed nicer) but for some reason unknown to me the script would complain it couldn't find the two config files. This seemed like a reasonable workaround which works.